### ISSUE TITLE
Add ttl as variable

### ISSUE THREAD

**Issue by rafaelkonrath** [terraform-aws-modules/terraform-aws-acm#34]
Hi, the current TTL set is hardcode in the main.tf, In my case I need to keep the TTL shorter than 60 since is this is not a variable I need to create a dependencies folder with the subject_alternative_names and change the TTL there.

**Comment by rafaelkonrath** [comment]
The way I'm using the module locally:

```
main.tf
-  ttl     = 60
+  ttl     = var.dns_ttl
```

```
variables.tf
+variable "dns_ttl" {
+  description = "The TTL of DNS recursive resolvers to cache information about this record."
+  type        = string
+  default     = "60"
+}
+
```

**Comment by bryantbiggs** [comment]
closed with #54
